src/msw/dcmemory.cpp

```diff
@@ -131,6 +131,16 @@ void wxMemoryDCImpl::DoSelect( const wxBitmap& bitmap )
     {
         m_oldBitmap = hBmp;
     }
+
+    // Remember content scale factor used by the bitmap: we don't use it
+    // ourselves, but this can be needed later for creating fonts of the
+    // correct size.
+    m_contentScaleFactor = bitmap.GetScaleFactor();
+    if ( m_contentScaleFactor != 1.0 )
+    {
+        // We need to choose the appropriately sized default font.
+        SetFont(wxNORMAL_FONT->Scaled(m_contentScaleFactor));
```

Comment:
Why not `GetFont().Scaled()`? And will a call to `GetFont()` already scale it?

Reply:
When I remove this, `wxDC` at 200% issue is fixed. It seems to double scale the font. What was the reason you added it?

Reply:
It didn't work for me without it, i.e. the font was still small in the screenshot, but I think I know why: I've tested this on a system with 2 displays, one at 100% and another at 200% DPI, after moving the sample window to the 200% DPI display. And in this case I suspect we do need to scale the font because it's created for the 100% display initially -- and would be wrong without `Scaled()`. I'm not sure how are we going to distinguish from the case when there is a single display using 200% DPI however...

Reply:
Actually we probably ought to reuse `WXAdjustToPPI()` here.